src/commands/preview.ts

```diff
@@ -127,11 +127,13 @@ export default async function preview(cliFlags: PreviewCliFlags) {
           return true; // ignore md or html files not in entries source
         }
         if (
-          config.themeIndexes.find((theme) =>
-            theme.type === 'file'
-              ? path === theme.destination
-              : theme.type === 'package' &&
-                pathStartsWith(path, theme.destination),
+          config.themeIndexes.find(
+            (theme) =>
+              (theme as any).destination !== theme.location &&
```

Comment:
Don't use `as any`. We can do type narrowing by `theme.type !== 'file'`.

```suggestion
              (theme.type !== 'file' || theme.destination !== theme.location) &&
```

Reply:
修正しました。

```ts
              (theme.type === 'file' || theme.type === 'package') &&
              theme.destination !== theme.location &&
              (theme.type === 'file'
                ? path === theme.destination
                : pathStartsWith(path, theme.destination)),
```

としました。 `theme.type === 'package'` の場合でも  `theme.destination !== theme.location` のチェックをするべきだと考えたためです。（#160 で「ローカル theme について、いま行っている theme のファイルのコピーを行わない」ようにする提案をしています。その場合 `theme.type === 'package'` の場合でも `theme.destination === theme.location` となります。）